bot/events/badbotHandler.js

```diff
@@ -0,0 +1,31 @@
+const _ = require('lodash');
+const source = require('rfr');
+
+const { sendMessage } = source('bot/utils/util');
+const logger = source('bot/utils/logger');
+
+const { Responses } = source('models/responses');
+const { Statistics, EventTypes } = source('models/statistics');
+
+const badBotRegex = /[Bb][Aa][Dd]\s+[Bb][Oo][Tt]/;
+
+function botMessage(message) {
+    if (message.guild === null || message.author.bot) return;
+
+    if (!badBotRegex.test(message.content)) return;
```

Comment:
Make lines 13 and 15 one line by introducing a function, ie.

```
function shouldHandleBotMessage(message) {
   if (message.guild === null || message.author.bot || !badBotRegex.test(message.content)) return true;
}
```

This would make it easier to understand, more modular, and easier to test later on